### ISSUE TITLE
Accessibility: alt attribute on wp-notification-image

### ISSUE THREAD

**Issue by joedolson** [WordPress/wp-feature-notifications#107]
The images associated with notifications should be considered decorative, but need an empty alt attribute to prevent screen readers from attempting to interpret them.

**Comment by erikyo** [comment]
closed with [#92](https://github.com/WordPress/wp-feature-notifications/pull/92)

https://github.com/WordPress/wp-feature-notifications/blob/680be57193c8f7cab73c48507b97696859ca1037/src/scripts/components/NoticeImage.js#L49
